CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
+
+Contributions are welcome! Please see the [development section](#development) below for more information. For new bpftrace tools, please add them to the new [user-tools repository](https://github.com/bpftrace/user-tools/blob/master/CONTRIBUTING.md). The tools that exist in this repository are a small collection curated by the bpftrace maintainers.
+
+* Bug reports and feature requests: [Issue Tracker](https://github.com/bpftrace/bpftrace/issues)
+
+* Development IRC: #bpftrace at irc.oftc.net
+
+* [Good first issues](https://github.com/bpftrace/bpftrace/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
+
+## Development
+
+* [Coding Guidelines](docs/coding_guidelines.md)
+* [Development Guide](docs/developers.md)
+* [Development Roadmap](https://docs.google.com/document/d/17729Rlyo1xzlJObzHpFLDzeCVgvwRh0ktAmMEJLK-EU/edit)
+* [Fuzzing](docs/fuzzing.md)
+* [Nix](docs/nix.md)
+* [Release Process](docs/release_process.md)
+* [Tests](tests/README.md)
+
+## Developer's Certificate of Origin
+
+Any contributions to bpftrace must only contain code that can legally be contributed to bpftrace, and which the bpftrace project can distribute under its license.
+
+Prior to contributing to bpftrace please read the [Developer's Certificate of Origin](https://developercertificate.org/)
+and sign-off all commits with the `--signoff` option provided by `git commit`. For example:
+
+```
+git commit --signoff --message "This is the commit message"
+```
+
+This option adds a `Signed-off-by` trailer at the end of the commit log message.
```

Comment:
I would also add few words here to make sure people use their real names in SOB instead of github login names or whatever else.